coderd/database/migrations/000255_workspace_agent_script_timings.up.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE workspace_agent_script_timings
```

Comment:
This table is nominally relating to `workspace_agents_scripts`, but we're not inferring any relationship to `workspace_agent_scripts` - which feels a bit odd.

Reply:
Hhmm, why the link to `workspace_agents` and not `workspace_agent_scripts`?

Reply:
`workspace_agent_scripts` has the following schema

```
       Column       |           Type           | Collation | Nullable | Default
--------------------+--------------------------+-----------+----------+----------
 workspace_agent_id | uuid                     |           | not null |
 log_source_id      | uuid                     |           | not null |
 log_path           | text                     |           | not null |
 created_at         | timestamp with time zone |           | not null |
 script             | text                     |           | not null |
 cron               | text                     |           | not null |
 start_blocks_login | boolean                  |           | not null |
 run_on_start       | boolean                  |           | not null |
 run_on_stop        | boolean                  |           | not null |
 timeout_seconds    | integer                  |           | not null |
 display_name       | text                     |           | not null | ''::text
```

I'm not sure which of these fields I could link to? Unless I create a new `id` field and link against that?

Reply:
Hah, was not expecting that. Yeah I think a new `id` is a good way forward 👍